arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.stm32f103xb

```diff
@@ -1,19 +0,0 @@
-# Kconfig - ST Microelectronics STM32F103RB MCU
```

Comment:
The arch/arm/soc/st_stm32/... Kconfig merging should be in the SoC patch, not the board one.